File: Python/response/mms/MMSMessageReceiptSubscriptionResponse.py
from response.ResourceReference import ResourceReference

class MMSMessageReceiptSubscriptionResponse:
	"""subscribe to notifications of MMS messages sent to your application"""
	
	def __init__(self):
		"""Default class constructor"""
		self.httpResponseCode=0
		self.contentType=None
		self.location=None
		self.httpResponseCode=0
		self.contentType=None
		self.location=None
		self.resourceReference=None
	
	def getHTTPResponseCode(self):
		"""getter for httpResponseCode : the numeric status code result of the HTTP response"""
		return self.httpResponseCode
	
	def setHTTPResponseCode(self,httpResponseCode):
		"""setter for httpResponseCode : the numeric status code result of the HTTP response"""
		self.httpResponseCode=httpResponseCode
	
	def getContentType(self):
		"""getter for contentType : the resulting MIME content type of the HTTP response"""
		return self.contentType
	
	def setContentType(self,contentType):
		"""setter for contentType : the resulting MIME content type of the HTTP response"""
		self.contentType=contentType
	
	def getLocation(self):
		"""getter for location : the contents of the HTTP response location header"""
		return self.location
	
	def setLocation(self,location):
		"""setter for location : the contents of the HTTP response location header"""
		self.location=location
	
	def getResourceReference(self):
		"""getter for resourceReference : the value of the resourceReference field"""
		return self.resourceReference
	
	def setResourceReference(self,resourceReference):
		"""setter for resourceReference : the value of the resourceReference field"""
		self.resourceReference=resourceReference
	
	def setResourceReferenceJSON(self,jsondata):
		self.resourceReference=ResourceReference(jsondata)
	

File: Ruby/response/mms/MMSMessageReceiptSubscriptionResponse.rb
require "response/ResourceReference"

class MMSMessageReceiptSubscriptionResponse

	def initialize
		@httpResponseCode=0
		@contentType=nil
		@location=nil
		@resourceReference=nil
	end

	def getHTTPResponseCode
		@httpResponseCode
	end

	def setHTTPResponseCode(httpResponseCode)
		@httpResponseCode=httpResponseCode
	end

	def getContentType
		@contentType
	end

	def setContentType(contentType)
		@contentType=contentType
	end

	def getLocation
		@location
	end

	def setLocation(location)
		@location=location
	end

	def getResourceReference
		@resourceReference
	end

	def setResourceReference(resourceReference)
		@resourceReference=resourceReference
	end

	def setResourceReferenceJSON(jsondata)
		@resourceReference=ResourceReference.new
		@resourceReference.initializeJSON(jsondata)
	end

end 
